packages/sdk-ui-ts/src/utils/mappings.ts

```diff
@@ -310,5 +310,7 @@ export const validatorAddressToPathMap = {
     injvalcons1m8shfyz7cu7w0umgwgmhk59nmatzlvu7g56t6a:
     'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp',
     injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj:
-    'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp'
+    'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp',
+    injvalcons1n5yluh6k9zqv74jw35r5s5zy808xpzfqp2wm7g:
+    'injvaloper1n5yluh6k9zqv74jw35r5s5zy808xpzfq4ea8jf.webp'
```

Comment:
You are still missing one mapping. Please fix it properly before committing again.

Reply:
Sorry for this, I have problem with my code editor app. Fixed from github